mbed_lib.json

```diff
@@ -2,7 +2,9 @@
     "name": "easy-connect",
     "target_overrides": {
         "*": {
-            "target.features_add": ["COMMON_PAL"]
+            "target.features_add": ["COMMON_PAL"],
+            "platform.stdio-baud-rate":115200,
+            "platform.stdio-convert-newlines":true
```

Comment:
Adding a README section on that. It should be done in apps, it's silly we have to do the \r\n.